contents/select.md

```diff
@@ -0,0 +1,191 @@
+## Select {#sec:select}
+
+Whereas `filter` removes rows, `select` removes columns.
+However, `select` is much more versatile than just removing columns, as we will discuss in this section.
+First, let's create a dataset with multiple columns:
+
+```jl
+@sco responses()
+```
+
+Here, the data represents answers on five questions (`q1`, `q2`, ..., `q5`) in a questionnaire.
+We will start by "selecting" a few columns from this dataset.
+Again, we use symbols to specify columns:
+
+```jl
+s = "select(responses(), :id, :q1)"
+sco(s, process=without_caption_label)
+```
+
+We can also use strings if we want:
+
+```jl
+s = """select(responses(), "id", "q1", "q2")"""
+sco(s, process=without_caption_label)
+```
+
+To select everything _except_ one or more columns, use `Not` with either a single column name:
+
+```jl
+s = """select(responses(), Not(:q5))"""
+sco(s, process=without_caption_label)
+```
+
+Or, with multiple column names:
+
+```jl
+s = """select(responses(), Not([:q4, :q5]))"""
+sco(s, process=without_caption_label)
+```
+
+It's also fine to combine column names with `Not`:
+
+```jl
+s = """select(responses(), :q5, Not(:id))"""
+sco(s, process=without_caption_label)
+```
+
+Note how `q5` is now the first column.
+But, there is a more clever way to do this, and that is with `:`.
+The colon `:` can be thought of as "all the columns that we didn't include yet".
+For example:
+
+```jl
+s = """select(responses(), :q5, :)"""
+sco(s, process=without_caption_label)
+```
+
+Or, to put `q5` at the second position[^sudete]:
+
+[^sudete]: Thanks to Sudete on [Discourse](https://discourse.julialang.org/t/pull-dataframes-columns-to-the-front/60327/4) for this suggestion.
+
+```jl
+s = "select(responses(), 1, :q5, :)"
+sco(s, process=without_caption_label)
+```
+
+Even renaming columns is possible via `select` via `source => target`:
+
+```jl
+s = """select(responses(), 1 => "participant", :q1 => "age", :q2 => "nationality")"""
+sco(s, process=without_caption_label)
+```
+
+which, thanks to the `...` operator, we can also write as:
```

Comment:
It is already the second time we are mentioning the splat operator `...`. I think I need to include it in the Julia chapter. What do you guys think? @lazarusA and @rikhuijzer

Reply:
Yes, I also use it a lot for some plots. 👍🏼

Reply:
Ok. gonna put in the TODO list with an issue and assign to me

Reply:
https://github.com/JuliaDataScience/JuliaDataScience/issues/31